File: Java/Dependency Inversion Principle/Kullanici.java
public class Kullanici {
    private String isim;
    private String mail;
    private String ip;
    private String tel;

    public Kullanici(String isim, String mail, String ip, String tel) {
        this.isim = isim;
        this.mail = mail;
        this.ip = ip;
        this.tel = tel;
    }

    public String getIsim() {return isim;}

    public void setIsim(String isim) {
        this.isim = isim;
    }

    public String getMail() {return mail;}

    public void setMail(String mail) {
        this.mail = mail;
    }

    public String getIp() {return ip;}

    public void setIp(String ip) {
        this.ip = ip;
    }

    public String getTel() {return tel;}

    public void setTel(String tel) {
        this.tel = tel;
    }
}

File: Python/Dependency Inversion Principle/Kullanici.py
class Kullanici:
    
    def __init__(self,isim=None,mail=None,ip=None,tel=None):
        self.isim=isim
        self.mail=mail
        self.ip=ip
        self.tel=tel
    
    def getİsim(self):
        return self.isim
    
    def setİsim(self,isim):
        self.isim=isim
        
    def getMail(self):
        return self.mail
    
    def setMail(self,mail):
        self.mail=mail
        
    def getİp(self):
        return self.ip
    
    def setİp(self,ip):
        self.ip=ip
        
    def getTel(self):
        return self.tel
    
    def setTel(self,tel):
        self.tel=tel
        